### PR TITLE
chore(Python): add program to print middle node element of linked list

### PR DESCRIPTION
**Pull request by makrandbhonde**
**PR Checklist:**
- [x] My submission is formatted according to the guidelines in the [contributing guide](https://github.com/MakeContributions/DSA#contribution-guidelines)
- [x] My addition is on refer on the language README.md file
- [x] My addition does not have a spelling problem
- [x] My submission has a proper and user-friendly description of the algorithm
- [x] My submission has the time complexity of the algorithm
- [x] My submission has sample input-output of the program (NOT FOR PYTHON)

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] New algorithm
- [ ] Optimization in previous algorithms
- [ ] Code style update
- [ ] Refactor
- [ ] Documentation
- [ ] Other, please describe:

**Briefly describe the changes in this PR**

**Other information:**

### CODE REVIEW

File: algorithms/Python/linked_lists/middle-node-linkedlist.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        self.head = new_node
+    
+    #Function returns data of middle node
+    def middle_element(self):
+        slow = self.head
+        fast = self.head
+        while fast and fast.next:
+            slow = slow.next
+            fast = fast.next.next
+        return slow.data
```
Comment (Arsenic-ATG) [inline algorithms/Python/linked_lists/middle-node-linkedlist.py:28]: what if the list is empty ?

Comment (makrandbhonde) [reply algorithms/Python/linked_lists/middle-node-linkedlist.py:28]: So if the list is empty we can add two lines of code in starting of the middle_element function :-
if self.head is None:
    return None

Comment (Arsenic-ATG) [reply algorithms/Python/linked_lists/middle-node-linkedlist.py:28]: indeed, that should do the trick.

Comment (makrandbhonde) [reply algorithms/Python/linked_lists/middle-node-linkedlist.py:30]: I have updated the code for empty list. Please check it and let me know

**Review by Arsenic-ATG — APPROVED** [review]

File: algorithms/Python/linked_lists/middle-node-linkedlist.py (hunk 1 above)
Comment (Arsenic-ATG) [reply algorithms/Python/linked_lists/middle-node-linkedlist.py:30]: Thanks you

**Pull request merged**